jsonexample/lib/built_value/built_complex_object.g.dart

```diff
@@ -7,7 +7,7 @@
 part of 'built_complex_object.dart';
 
```

Comment:
Who generates this?

Reply:
It's from the `built_value` package. I've managed to double up some commits in this PR. This is from our JSON example app.